src/Core/Managed/Net40/Extensibility/Implementation/TelemetryConfigurationFactory.cs

```diff
@@ -44,7 +48,7 @@ public static TelemetryConfigurationFactory Instance
             set { instance = value; }
         }
 
-        public virtual void Initialize(TelemetryConfiguration configuration)
+        public virtual void Initialize(TelemetryConfiguration configuration, IList<ITelemetryModule> modules)
```

Comment:
Can we pass TelemetryModules class instead of using ```IList``` directly? We can change implementation of it later this way